test/testrunner.py

```diff
@@ -442,9 +442,14 @@ def main():
             sys.exit(pytest.main(pytest_cmd))
 
         else:
-            run_sagemaker_remote_tests(
-                [image for image in standard_images_list if not ("tensorflow-inference" in image and "py2" in image)]
-            )
+            if "neuron" in dlc_images:
+                run_sagemaker_remote_tests(
+                    [image for image in standard_images_list if ("mxnet" in image and "1.8.0" in image)]
```

Comment:
Could you please remove this string-matching check for "1.8.0" in the image URIs being passed for testing?

This introduces issues when there are new neuron images built for future framework versions, and also causes code to be duplicated between the two if-else conditions every time there needs to be a change.

Instead, please move the framework version check to within the conftest.py for SM MXNet Inference tests for Neuron images, and change the implementation to `packaging.version.Version(framework_version) >= packaging.version.Version("1.8")` so that there is no code change that needs to happen for future versions.